blocks/footer/footer.js

```diff
@@ -11,7 +11,7 @@ import {
 function setupCookieChoices(section) {
   const cookieLink = section.querySelector('a[href*="onetrust-link"]');
   if (cookieLink) {
-    cookieLink.removeAttribute('href');
+    cookieLink.setAttribute('href', '#');
```

Comment:
This appears to break the Cookie Preferences link. Should the `a` be transformed into a button?

Reply:
Good catch! Addressed in [e518c84](https://github.com/hlxsites/pga-presidents-cup/pull/3/commits/e518c84c356988cfd044cc965eff047b81a22bd9)